spec/models/concerns/hyrax/collection_behavior_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.describe Hyrax::CollectionBehavior, clean_repo: true do
+  let(:collection) { create(:collection_lw) }
+  let(:work) { create(:work) }
+
+  describe "#destroy" do
+    # A little bit about this spec.  It's informed by
```

Comment:
i'd prefer to drop this long comment, since it's in the commit message and this test is relevant outside of the context of the particular regression it guards.